ui/imports/shared/views/PasswordView.qml

```diff
@@ -14,14 +14,13 @@ import StatusQ.Components 0.1
 ColumnLayout {
     id: root
 
-    property bool ready: newPswInput.text.length >= root.minPswLen && newPswInput.text === confirmPswInput.text && errorTxt.text === ""
-    property int minPswLen: 10
```

Comment:
WDYT on just keeping this `minPswLen` property here and assign the `Constants.minPasswordLength` instead?

Reply:
Yes, this was my first intention. I was thinking to reuse it in order to lower the regression risk. But I didn't like the fact that is part of the public API. Moving it to the private obj would require the same amount of changes and it's not improving the regression risk.
If you think it's worth keeping it we could change it to readonly and keep it.

Reply:
I don't think it is a bad option keeping the public property so you are giving the possibility to the consumer to change the length, but you have defined a `by default` constant value. 

It is just a suggestion, I leave the final decision up to you. :)